### ISSUE TITLE
detecting chinese in images

### ISSUE THREAD

**Issue by chajbn** [MhLiao/TextBoxes#54]
I have tried to apply the framework of TextBoxes to detect chinese in images,however it doesn't have a good result.Is there any suggestions for improvement?

**Comment by MhLiao** [comment]
Did you train it on Chinese data?

**Comment by chajbn** [comment]
Yes,I have trained it with the initial weight of the TextBoxes-icdar13.caffemodel on about 8k images with chinese.After about 40000 iterations with base_lr  0.001,the final loss is about 2 and the visual result is bad.Do you have done the same work?

**Comment by InesBeltaief** [comment]
hey @chajbn I'm trying to use TextBoxes in my custom data but i have no idea about how to do it, could you please help me with the steps of training you followed to train chinese dataset ?

**Comment by dajiangxiaoyan** [comment]
@chajbn I also met the same problem. When training my own Chinese scene text image, the final loss is about 1.5. And do you have any suggestion?

**Comment by chajbn** [comment]
For me, I think it is mainly because that the  Chinese text  in my scene images is too disordered to get a good result quickly. You can have a look at your training dataset.

**Comment by DemonProfessor** [comment]
@https://github.com/dajiangxiaoyan
I'm sry to disturb you. I tried to train a new model with my own data , and now i have got the lmdb .But i don't know how to cteate the train.prototxt and what to do next. could u please give  me some guidance if you could see the message.  And my QQ is 646803015

**Comment by liuyiyiyiyi** [comment]
> I have tried to apply the framework of TextBoxes to detect chinese in images,however it doesn't have a good result.Is there any suggestions for improvement?

Have you imporoved the accuracy?I train the chinese detetion model on my dataset,but the map is only o.23.

**Comment by liuyiyiyiyi** [comment]
Have you improved the accuracy?I train the chinese detetion model on my dataset,but the map is only o.23.
